### DISCUSSION TITLE
How do I listen and get the response from `forceReply` of a message?

### DISCUSSION THREAD

**Discussion by hashfi-motion** [telegraf/telegraf#1932] [Q&A] [answered]
As the title said. How do I listen and get the response from a message with markup `forceReply`?

I mean I have an idea how, by doing `.on('text', ...)` and making condition to check if the `id` is the same of the message with `forceReply` markup. But is there any better and _correct_ way to do this? Like already built in method on the `ctx`?

**Comment by MKRhere** [comment, marked as answer]
There isn't a builtin method. You already have the right idea.
